datadog_checks_dev/datadog_checks/dev/utils.py

```diff
@@ -182,6 +183,24 @@ def get_here():
     return get_parent_dir(inspect.currentframe().f_back.f_code.co_filename)
 
 
+def load_jmx_config():
+    root = get_parent_dir(inspect.currentframe().f_back.f_code.co_filename)
```

Comment:
Could this use the new get_here method you made?

Reply:
No I think b/c that would get this location

Reply:
Ahh good point